testing/BUILD

```diff
@@ -66,3 +85,24 @@ declare_deps_provider(
         "//external:io_bazel_rules_scala/dependency/scala/scalatest/scalatest",
     ],
 )
+
+declare_deps_provider(
+    name = "specs2_classpath_provider",
+    deps_id = "specs2_classpath",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@io_bazel_rules_scala_org_specs2_specs2_common",
```

Comment:
Why all other deps are in form `//external:io_bazel_rules_scala/dependency/...` and these are not?

Reply:
It comes from the existing code: https://github.com/bazelbuild/rules_scala/blob/master/specs2/BUILD#L9

It's binded later in https://github.com/bazelbuild/rules_scala/blob/master/specs2/specs2.bzl#L26

Existing binds are subject to be removed later. Currently the goal is to give a way to not use binds.